tools/alphafold/alphafold.xml

```diff
@@ -56,10 +56,11 @@
 
         ## generate extra outputs -----------------
         ## plddts
-        python3 $__tool_directory__/gen_extra_outputs.py plddts &&
+        python3 '$__tool_directory__/gen_extra_outputs.py' plddts &&
         mkdir -p '${ html.files_path }' &&
 
         ## html
+	## that should probably be shipped with the tool

```

Comment:
Can we ship this file with the tool, or is it too big?

Reply:
We definitely can, just were using this approach during the development process. Will still perform a few tweaks over the next few days, but will include the file when its placed on the toolshed.